ports/zeroptianinvasion/gameinfo.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<gameList>
+  <game>
+    <path>./Zeroptian Invasion.sh</path>
+    <name>Zeroptian Invasion</name>
+    <desc>Zeroptian Invasion is a classic shoot 'em up with mechanics from other arcades of the 80s. Travel back in time with this unique pixelated experience featuring an incredible soundtrack in chiptune.</desc>
+    <releasedate>20181128T000000</releasedate>
+    <developer>Joysan</developer>
+    <publisher>Self-Published</publisher>
+    <genre>Arcade</genre>
+    <image>./zeroptianinvasion/screenshot.png</image>
```

Comment:
![zeroptianinvasion-fs8](https://github.com/user-attachments/assets/56a0b392-03ec-4986-b95b-6554178ddb61)
What do you think ? :)

Reply:
Thanks. Looks good. Added as cover.png